types/datum.go

```diff
@@ -1501,6 +1501,8 @@ func (d *Datum) ToFloat64(sc *stmtctx.StatementContext) (float64, error) {
 // ToString gets the string representation of the datum.
 func (d *Datum) ToString() (string, error) {
 	switch d.Kind() {
+	case KindNull:
```

Comment:
https://github.com/pingcap/tidb/pull/6647
You may need to discuss with @coocood